glide.lock

```diff
@@ -131,9 +131,11 @@ imports:
 - name: github.com/cznic/mathutil
   version: 250d0b9d3304c5ea0c4cfc7d9efc7ee528b81f3b
 - name: github.com/cznic/ql
-  version: 77545ff365ff426dfab06c46d70fa77600cd55af
+  version: 2cbfdfc32e5e9575be5176b1f6f21d6445003617
   subpackages:
   - driver
+  - vendored/github.com/camlistore/go4/lock
+  - vendored/github.com/cznic/exp/lldb
```

Comment:
Unfortunately there is not much we can do here, since ql does import rewriting for those "vendored" packages, see https://github.com/cznic/ql/blob/master/file.go#L23-L24.